plugins/plugin-site/src/components/PluginHealthScoreProgressBar.jsx

```diff
@@ -1,11 +1,13 @@
 import React, {useState} from 'react';
 import PropTypes from 'prop-types';
 import {Progress, Tooltip} from 'reactstrap';
+ import '../styles/progressbar.css';
```

Comment:
can you change this to a css module, so we don't introduce global styles with generic names?